cli/spcli/actor.go

```diff
@@ -265,6 +265,10 @@ func ActorSetAddrsCmd(getActor ActorAddressGetter) *cli.Command {
 
 			ctx := lcli.ReqContext(cctx)
 
+			if cctx.NArg() != 1 {
```

Comment:
too late, but it's already handled at the top, see the way it deals with `args`, let's remove this one